ui/src/components/Item/Detail.svelte

```diff
@@ -99,15 +100,44 @@
             by {patp}
           </a>{/if}
       </div>
-      {#if window.ethereum && me !== patp}
-        <div class="flex">
+      <div class="flex gap-3">
+        {#if window.ethereum && me !== patp}
           <IconButton
             icon={EthereumIcon}
-            class="border"
+            class="bg-panels dark:bg-transparent hover:bg-panels-hover dark:hover:border-white dark:border border"
             on:click={() => handleTipRequest(key)}>Tip</IconButton
           >
-        </div>
-      {/if}
+        {/if}
+        {#if !isInstalledOrJoined}
+          {#if type === 'group'}
+            <div class="flex md:hidden">
+              <IconButton
+                icon={PlusIcon}
+                on:click={async (event) => {
+                  event.target.innerHTML = 'Joining';
+                  await api.urbit.do.joinGroup(key).then(refreshGroups);
+                }}
+                class="bg-panels dark:bg-transparent hover:bg-panels-hover dark:hover:border-white dark:border border"
+                >Join Group</IconButton
+              >
+            </div>
+          {/if}
+          {#if type === 'app'}
+            <div class="flex md:hidden">
+              <IconButton
+                icon={InstallIcon}
+                on:click={(event) => {
+                  window.open(
+                    `${window.location.origin}/apps/grid/search/${patp}/apps`
+                  );
+                }}
+                class="bg-panels dark:bg-transparent hover:bg-panels-hover dark:hover:border-white dark:border border"
+                >Join Group</IconButton
+              >
+            </div>
+          {/if}
+        {/if}
+      </div>
```

Comment:
I'm a bit confused I thought these features existed already?

Reply:
they live in a sidebar group that's invisible on mobile. trying to rearrange that sidebar group to look good on mobile was proving to be a pain, so this was close enough

Reply:
moved here https://github.com/worpet-bildet/portal/pull/155